### ISSUE TITLE
Some meters are not shown

### ISSUE THREAD

**Issue by evshvarov** [intersystems-community/DeepSeeWeb#348]
[Example](https://pie-chart-bug.demo.community.intersystems.com/dsw/index.html#/USER/Bug/Meter%20Names.dashboard)

**Comment by jakcpto** [comment]
@gnibeda hello. Backend return all necessary data, I think.
in Widget section of Dashboard method

**Comment by jakcpto** [comment]
Here is screenshot
<img width="890" alt="image" src="https://user-images.githubusercontent.com/41373877/220255734-befcf0a1-92fd-40b5-a3f3-6b55314c4b1c.png">

**Comment by Lena-Ev** [comment]
for every DataValue find its Dimension ??
<img width="1434" alt="image" src="https://user-images.githubusercontent.com/77621970/220329984-bf4d941c-dae6-4074-bae0-18f0ebc777ee.png">

**Comment by Frisle** [comment]
Here's what I found. The state of the pie-chart and my local copy of samples-bi are different when it comes to text meters, although the mdx2json and dsw versions are the latest. I changed the way how ResultSet is taking dimension name. Now it comes directly from the axes.
So, there is the state of my copy.

Before fix all meters were broken down
![chrome_m5fu1biIGa](https://user-images.githubusercontent.com/47441164/220569943-54c64aef-a096-4820-a820-2083a3152f93.png)

After fix meters displayed as expected. And any other meters are still working after that.
![chrome_ZqHNuQt23x](https://user-images.githubusercontent.com/47441164/220570450-c6feb183-6d1f-4ad7-913a-b40ad2bca4ae.png)
